guides/common/modules/ref_provisioning-settings.adoc

```diff
@@ -0,0 +1,74 @@
+This permits users full remote code execution on {ProjectServer}, effectively disabling all authorization.
+This is not a safe option, especially in larger companies.
+| *Access unattended without build* | No | Enable access to unattended URLs without build mode being used.
+| *Query local nameservers* | No | {Project} queries the locally configured resolver instead of the SOA/NS authorities.
+| *Installation token lifetime* | 360 | Time in minutes that installation tokens should be valid for.
+Set to 0 to disable the token.
+| *SSH timeout* | 120 | Time in seconds before SSH provisioning times out.
+| *Libvirt default console address* | 0.0.0.0 | The IP address that should be used for the console listen address when provisioning new virtual machines using libvirt.
+| *Update IP from built request* | No | {Project} updates the host IP with the IP that made the build request.
+| *Use short name for VMs* | No | {Project} uses the short hostname instead of the FQDN for creating new virtual machines.
+| *DNS timeout* | [5, 10, 15, 20] | List of timeouts (in seconds) for DNS lookup attempts such as the `dns_lookup` macro and DNS record conflict validation.
+| *Clean up failed deployment* | Yes | {Project} deletes the virtual machine if the provisioning script ends with a non-zero exit code.
+| *Type of name generator* | `Random-based` | Specifies the method used to generate a hostname when creating a new host.
+
+The default `Random-based` option generates a unique random hostname which you can but do not have to use.
+This is useful for users who create many hosts and do not know how to name them.
+
+The `MAC-based` option is for bare-metal hosts only.
+If you delete a host and create it later on, it receives the same hostname based on the MAC address.
+This can be useful for users who recycle servers and want them to always get the same hostname.
+
+The `Off` option disables the name generator function and leaves the hostname field blank.
+| *Default PXE global template entry* | | Default PXE menu item in a global template - `local`, `discovery` or custom, use blank for template default.
+| *Default PXE local template entry* | | Default PXE menu item in local template - `local`, `local_chain_hd0`, or custom, use blank for template default.
+| *iPXE intermediate script* | iPXE intermediate script | Intermediate iPXE script for unattended installations.
+| *Destroy associated VM on host delete* | No | Destroy associated VM on host delete.
+When enabled, VMs linked to hosts are deleted on Compute Resource.
+When disabled, VMs are unlinked when the host is deleted, meaning they remain on Compute Resource and can be re-associated or imported back to {Project} again.
+This does not automatically power off the VM
+| *Maximum structured facts* | 100 | Maximum number of keys in structured subtree, statistics stored in {project-context}::dropped_subtree_facts.
```

Comment:
Will the attribute be replaced if we code format `{project-context}::dropped_subtree_facts` ?